### PR TITLE
feat(suite-desktop): bundler for electron source

### PR DESCRIPTION
**Pull request by keraf**
This adds `esbuild` as a bundler for the Electron source which now outputs two files (`electron.js` and `preload.js`). We can also import resources from outside the `src-electron` folder without messing up the output. An example is provided with importing `TOR_DOMAIN` from the `@trezor/suite` project.

### CODE REVIEW

File: packages/suite-desktop/scripts/build.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
```
Comment (szymonlesisz) [inline packages/suite-desktop/scripts/build.js]: i would rater disable one rule (require) than the whole eslint

File: packages/suite-desktop/scripts/dev.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable */
```
Comment (szymonlesisz) [inline packages/suite-desktop/scripts/dev.js]: i would rater disable one rule (require) than the whole eslint

**Review by szymonlesisz — CHANGES_REQUESTED** [review]
looks good, just eslint nitpicks

File: packages/suite-desktop/scripts/build.js (hunk 1 above)
Comment (keraf) [reply packages/suite-desktop/scripts/build.js]: That was just to get my IDE to stop complaining about these files (which shouldn't be linted). Added the `scripts` folder to the eslint ignore. 

3d8fd3b

File: packages/suite-desktop/scripts/dev.js (hunk 2 above)
Comment (keraf) [reply packages/suite-desktop/scripts/dev.js]: 3d8fd3b

**Review by szymonlesisz — APPROVED** [review]

**Comment by keraf** [comment]
Woops, I just realized that I accidentally left a webpack configuration file (while playing with other bundlers) in this commit. No impact, will be deleted from develop.

**Pull request merged**